contrib/src/test/resources/RegexSplitterschema.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
@devtagare It doesn't need the delimiters in schema definition, will remove them from the test schema.